web/src/admin/stages/email/EmailStageForm.ts

```diff
@@ -234,6 +234,30 @@ export class EmailStageForm extends BaseStageForm<EmailStage> {
                             })}
                         </select>
                     </ak-form-element-horizontal>
+                    <ak-form-element-horizontal
+                        label=${msg("Account Recovery Max Attempts")}
+                        required
+                        name="recoveryMaxAttempts"
+                    >
+                        <input
+                            type="number"
+                            value="${this.instance?.recoveryMaxAttempts ?? 5}"
+                            class="pf-c-form-control"
+                            required
+                        />
+                    </ak-form-element-horizontal>
+                    <ak-form-element-horizontal
+                        label=${msg("Account Recovery Cache Timeout")}
+                        required
+                        name="recoveryCacheTimeout"
+                    >
+                        <input
+                            type="number"
+                            value="${this.instance?.recoveryCacheTimeout ?? 300}"
+                            class="pf-c-form-control"
+                            required
+                        />
+                    </ak-form-element-horizontal>
```

Comment:
The form is not sending the values for `Account Recovery Max Attempts` nor `Account Recovery Cache Timeout` 
![Jul-23-2025 05-38-06](https://github.com/user-attachments/assets/f955df87-d0f2-41ec-a7f8-5dacd26ac71e)